app/models/user.rb

```diff
@@ -98,25 +91,10 @@ def reset_2fa!
       encrypted_otp_secret_key_salt: nil,
       totp_timestamp: nil,
       otp_secret_key: nil,
-      second_factor_method: nil,
     )
   end
 
   def send_new_otp_after_login?
```

Comment:
`send_new_otp_after_login` is called from the 2fa Gem and is used to send an email to the user in case of email based 2fa. Since we don't do that it has to be set to false.

https://github.com/Houdini/two_factor_authentication/blob/7e0faa6e1f3ab497767c59e1e97fb01e340b59ce/lib/two_factor_authentication/hooks/two_factor_authenticatable.rb#L10